File: python/exercises/swapping.py
#Function to swap the numbers
def swap(x, y):
    temp = x    
    x = y
    y = temp
    print(x, y)
#main()
x=float(input("Give the first: "))
y=float(input("Give the second: "))
swap(x, y)
#end

File: C/exercises/swapping.c
#include <stdio.h>

void swap(int *a, int *b){
    int temp = *a;
    *a = *b;
    *b = temp;
}

int main(){
    int a, b;

    printf("Give the first number: ");
    scanf("%d", &a);

    printf("Give the second number: ");
    scanf("%d", &b);

    printf("Before swap: a = %d, b = %d\n", a, b);

    swap(&a, &b);

    printf("After swap: a = %d, b = %d\n", a, b);

    return 0;
}
